### PR TITLE
Update shadow-roles.md

### LINKED ISSUE #4591: fixing shadow-roles.md
I was in the middle of a review and someone lgtm'ed it. I had some comments that need to be addressed in a new PR. 

https://github.com/kubernetes/community/pull/4574

[this doc](https://github.com/kubernetes/community/blob/master/mentoring/programs/shadow-roles.md)

Screen Shot 2020-03-10 at 9 57 58 AM

### PR DESCRIPTION
**Pull request by zestrells**
Signed-off-by: zestrells zjestrella1@gmail.com

Which issue(s) this PR fixes:
Fixes #4591

Some comments were not addressed for issue #4591 like:

- Update and add a description of the Release Team
- Add a link to an overview of the Release Team
- Adding a line to Contributor Summit Planning

@markyjackson-taulia @parispittman

### CODE REVIEW

File: mentoring/programs/shadow-roles.md [hunk 1]
```diff
@@ -4,15 +4,16 @@ This area lists all of the shadow programs and how to get involved.
...
+
+An overview of the release team can be found [here](https://github.com/kubernetes/sig-release/tree/master/release-team)
 
 The list of roles and duties can be found [here](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks)
 
 ## API Review
 
 ## Contributor Summit Planning
-The Content subteam of the events team is responsible for programming and content for periods of the contributor summit.
+The Content subteam creates in person and virtual experiences for new and current contributers to come learn, share and collaborate.
```
Comment (markyjackson-taulia) [inline mentoring/programs/shadow-roles.md]: `contributers` is misspelled. It should be `contributors`. The test is failing for this. Please correct, push changes (don't forget to sign the commit.

Comment (zestrells) [reply mentoring/programs/shadow-roles.md]: Does every commit need to be signed, or just the initial one?

**Comment by markyjackson-taulia** [comment]
Congratulations 👌👍✌️

File: mentoring/programs/shadow-roles.md (hunk 1 above)
Comment (markyjackson-taulia) [reply mentoring/programs/shadow-roles.md]: Every commit

Comment (zestrells) [reply mentoring/programs/shadow-roles.md]: Ahh ok, will do so for next time. Sorry about that!

File: mentoring/programs/shadow-roles.md [hunk 2]
```diff
@@ -4,15 +4,16 @@ This area lists all of the shadow programs and how to get involved.
 Some areas are still in the process of forming shadow roles. 
 
 ## Release Team
-This SIG adheres to the Roles and Organization Management outlined in sig-governance and opts-in to updates and 
-modifications to sig-governance.
+This SIG release team is embedded within SIG Release and is responsible for the day to day work required to successfully release while the SIG at large is focused on the continued improvement of the release process.
+
+An overview of the release team can be found [here](https://github.com/kubernetes/sig-release/tree/master/release-team)
```
Comment (nikhita) [inline mentoring/programs/shadow-roles.md]: nit: we generally prefer to use git.k8s.io links, can you update this to:

```suggestion
An overview of the release team can be found [here](https://git.k8s.io/sig-release/release-team).
```

**Comment by markyjackson-taulia** [comment]
@zestrells i can help you (if you need that) tomorrow

**Comment by mrbobbytables** [comment]
If you could, please squash your commits down 👍 After that should be good to go!

**Comment by nikhita** [comment]
Thanks @zestrells!

/approve

**Pull request merged**